.github/ISSUE_TEMPLATE/release_tracker.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Release Tracker Issue"
+description: "Track component releases and update manifests before the deadline"
+title: "[Release Tracker] - <Insert Release Name>"
+body:
+  - type: input
+    id: release_date
+    attributes:
+      label: "📅 Planned Release Date"
+      description: "Enter the planned release date (e.g., Mar 21st, 2025)"
+      placeholder: "Mar 21st, 2025"
+
+  - type: input
+    id: deadline_date
+    attributes:
+      label: "🛑 Manifest Update Deadline"
+      description: "Enter the last date for updating manifests (e.g., Mar 14th, 2025)"
+      placeholder: "Mar 14th, 2025"
```

Comment:
same here, why need placeholder?

Reply:
it is Sunday (8th June as code freeze) and Monday (9th June release ODH)?

Reply:
😆 my bad I was thinking today as tuesday